ush/radmon_verf_angle.sh

```diff
@@ -176,9 +176,10 @@ EOF
 
 	 startmsg
          ./${angle_exec} < input >> "${pgmout}" 2>>errfile
-         export err=$?; err_chk
+         export err=$?
          if [[ ${err} -ne 0 ]]; then
-             fail=$(( fail + 1 ))
```

Comment:
Remove `fail` later in script?

Reply:
Yes, it should be.

That said, I realize now that `rstprod.sh` will not be called.  I think I will move it's call to the ex-script(s) instead.

Reply:
Fixed and refactored